src/cmdstan/command.hpp

```diff
@@ -594,6 +594,149 @@ int command(int argc, const char *argv[]) {
     return_code = stan::services::standalone_generate(
         model, fitted_params.samples.block(0, meta_cols, num_rows, num_cols),
         random_seed, interrupt, logger, sample_writers[0]);
+  } else if (user_method->arg("log_prob")) {
+    string_argument *upars_file = dynamic_cast<string_argument *>(
+        parser.arg("method")->arg("log_prob")->arg("unconstrained_params"));
+    string_argument *cpars_file = dynamic_cast<string_argument *>(
+        parser.arg("method")->arg("log_prob")->arg("constrained_params"));
+    bool jacobian_adjust
+        = dynamic_cast<bool_argument *>(
+              parser.arg("method")->arg("log_prob")->arg("jacobian_adjust"))
+              ->value();
+    if (upars_file->is_default() && cpars_file->is_default()) {
+      msg << "No input parameters provided, cannot calculate log-probability";
```

Comment:
log-probability -> log probability

P.S.  The ridiculous rule of English punctuation is that you hyphenate noun compounds used as adjectives (e.g., "log-probability calculation") but not noun compounds used as nouns (e.g. "we calculate the log probability").

Also, this should say "log probability density".